### PR TITLE
Let all collectibles fly to player when level is cleared

### LINKED ISSUE #47: transition: let collectibles fly to player when level is cleared

### LINKED ISSUE #55: transition: Joystick stuck when going to next level
When the player clears the level while dragging the joystick in a certain direction, the pause effect causes the joystick to be stuck in that position.

Then after the player reaches the next level, he is immediately propelled in the opposite direction because he continues shooting in that direction.

image

### PR DESCRIPTION
**Pull request by daniellimws**
Also change Level1 to have only one wave with one mob to make testing easier

Can help:
1. Check see if ok if just speed up the collectible movement by 5 times? Or want more fancy formula?
2. Help reject input when this is happening (just change and push to this branch)

### CODE REVIEW

**Comment by daniellimws** [comment]
Do we wanna let the crates auto-destroy themselves when the enemies are dead? Or let the player destroy all crates which imo is quite pointless?

**Review by jeffsieu — CHANGES_REQUESTED** [review]
Refactor disabled to enabled

File: Loot/Collectible.gd [hunk 1]
```diff
@@ -32,7 +33,15 @@ func set_value(value: int) -> void:
 func _physics_process(delta: float) -> void:
 	if _is_flying_to_player:
 		var distance_to_player = (_player.transform.origin - transform.origin).length()
-		velocity += (_player.transform.origin - transform.origin).normalized() / pow(distance_to_player, 2) * 50
+		var acceleration: Vector3 = (_player.transform.origin - transform.origin).normalized() / pow(distance_to_player, 2)
```
Comment (jeffsieu) [inline Loot/Collectible.gd]: Can change this to `transform.origin.direction_to(_player.transform.origin)`?

**Review by jeffsieu — CHANGES_REQUESTED** [review]

File: Loot/Collectible.gd (hunk 1 above)
Comment (jeffsieu) [reply Loot/Collectible.gd]: And `var distance_to_player = ...distance_squared_to(..)`

**Review by jeffsieu — APPROVED** [review]

**Pull request merged**